sdk/mobile-center-distribute/src/main/java/com/microsoft/azure/mobile/distribute/ErrorDetails.java

```diff
@@ -0,0 +1,48 @@
+package com.microsoft.azure.mobile.distribute;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+/**
+ * Release details JSON schema.
+ */
+class ErrorDetails {
```

Comment:
If the code is not one we know, the full payload is displayed already in the error. Why would we parse it in that case?